src/components/Viticulture/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import { times } from "lodash";
-import WineGlass from "../WineGlass/WineGlass";
+import WineGlass from "components/WineGlass";
```

Comment:
previously we discussed, maybe, moving "generic" components like `WineGlass` into the realms to which they apply. not sure this 40-file PR makes sense as the right place for it, but maybe sometime in the future.